content/applications/inventory_and_mrp/inventory/management/warehouses/create_a_second_warehouse.rst

```diff
@@ -0,0 +1,96 @@
+- :guilabel:`Manufacture`: select the route that should be followed when manufacturing goods inside
+  the warehouse
+- :guilabel:`Buy to Resupply`: check the box to allow for purchased products to be delivered to the
+  warehouse
+- :guilabel:`Resupply From`: select warehouses that can be used to resupply the warehouse being
+  created
+
+.. image:: create_a_second_warehouse/new_warehouse_configuration.png
+  :align: center
+  :alt: A filled out form for creating a new warehouse.
+
+.. important::
+  Creating a second warehouse will automatically enable the *Storage Locations* setting, which
+  allows location tracking of products within a warehouse. To toggle this setting, navigate to
+  :menuselection:`Configuration --> Settings` and click the checkbox under the *Warehouse* heading.
+
+After filling out the form, click :guilabel:`Save` and the new warehouse will be created.
+
+Add inventory to a new warehouse
+================================
+
+If a new warehouse is created that has existing inventory, this should be added to Odoo so that
+overall product counts are accurate. To add inventory to a new warehouse, start from
+:guilabel:`Inventory` and select :menuselection:`Operations --> Inventory Adjustments --> Create`,
+then fill out the inventory adjustment form as follows:
+
+- :guilabel:`Inventory Reference`: the name or code that the inventory adjustment can be referred to
+  by
+- :guilabel:`Locations`: the location or locations where the inventory is stored; include the new
+  warehouse and any locations within it that inventory will be added to
+- :guilabel:`Products`: include all products that will be added to inventory or leave blank to
+  select any product during the next step
+- :guilabel:`Include Exhausted Products`: include products with a quantity of zero; does not affect
+  inventory adjustments for new warehouses since they have no existing inventory
+- :guilabel:`Accounting Date`: the date used by accounting teams for bookkeeping related to the
+  inventory
+- :guilabel:`Company`: the company that owns the inventory; can be set as the user's company or as a
+  customer or vendor
+- :guilabel:`Counted Quantities`: choose whether the counted quantities for products being added
+  should default to stock on hand or zero; does not affect inventory adjustments for new warehouses
+  since they have no existing inventory
+
+.. image:: create_a_second_warehouse/inventory_adjustment_configuration.png
+  :align: center
+  :alt: A filled out form for an inventory adjustment.
+
+Once the form is properly configured, click on :guilabel:`Start Inventory` to be taken to the next
+page where products can be added to the inventory adjustment. Add a new product by clicking on
+:guilabel:`Create` and then fill out the product line as follows:
+
+- :guilabel:`Product`: the product being added to inventory
+- :guilabel:`Location`: the location where the product is stored; this can be the new warehouse or a
```

Comment:
Which location: the _old_ location where the stock was previously before being shifted to the new warehouse, or the _new_ location inside the additional warehouse?